assets/src/components/component/ComponentDetails.tsx

```diff
@@ -174,44 +181,58 @@ export function ComponentDetails({
       }
       heading={componentName}
       headingContent={
-        <div
-          css={{
-            display: 'flex',
-            gap: theme.spacing.medium,
-            className: 'DELETE',
-            margin: `${theme.spacing.medium}px 0`,
-          }}
-        >
-          <TabList
-            gap="xxsmall"
-            stateRef={tabStateRef}
-            stateProps={{
-              orientation: 'horizontal',
-              selectedKey: currentTab?.path,
+        <div>
+          <div
+            css={{
+              display: 'flex',
+              gap: theme.spacing.medium,
+              className: 'DELETE',
+              margin: `${theme.spacing.medium}px 0`,
             }}
           >
-            {filteredDirectory.map(({ label, path }) => (
-              <LinkTabWrap
-                key={path}
-                textValue={label}
-                to={path}
-                subTab
+            <TabList
+              gap="xxsmall"
+              stateRef={tabStateRef}
+              stateProps={{
+                orientation: 'horizontal',
+                selectedKey: currentTab?.path,
+              }}
+            >
+              {filteredDirectory.map(({ label, path }) => (
+                <LinkTabWrap
+                  key={path}
+                  textValue={label}
+                  to={path}
+                  subTab
+                >
+                  <SubTab>{label}</SubTab>
+                </LinkTabWrap>
+              ))}
+            </TabList>
+            {serviceDeploymentServiceId && service?.cluster?.id && (
+              <Button
+                as={Link}
+                to={getServiceDetailsPath({
+                  serviceId: serviceDeploymentServiceId,
+                  clusterId: service?.cluster?.id,
```

Comment:
This is not guaranteed to be the cluster actually.  If we need both you can actually load the actual `ServiceDeployment` by querying the `reference` object on the service

Reply:
A service crd is usually created on the management cluster, but will be used to create services on any other cluster